flags.go

```diff
@@ -202,6 +202,57 @@ func (f *Flags) String(short, long, defaultValue, help string) {
 		})
 }
 
+// StringSlice registers a stringSlice flag.
+func (f *Flags) StringSlice(short, long string, defaultValue []string, help string) {
+	f.register(short, long, help, "stringSlice", true, defaultValue,
+		func(res FlagMap) {
+			res[long] = &FlagMapItem{
+				Value:     []interface{}{},
+				IsDefault: true,
+			}
+			for _, v := range defaultValue {
+				res[long].Value = append(res[long].Value.([]interface{}), v)
+			}
+
+		},
+		func(flag, equalVal string, args []string, res FlagMap) ([]string, bool, error) {
+			if !f.match(flag, short, long) {
+				return args, false, nil
+			}
+			if len(equalVal) > 0 {
+				if res[long] != nil {
+					res[long].Value = append(res[long].Value.([]interface{}), trimQuotes(equalVal))
```

Comment:
this branch is unnecessary. do a `if res[long] == nil` check and initialize the flagmapitem there. then afterwards, add the equalval

Reply:
done